main.go

```diff
@@ -66,6 +66,13 @@ var (
 	resyncInterval = flag.Int("resync-interval", 30, "Informer resync interval in seconds")
 	namespace      = flag.String("namespace", apiv1.NamespaceAll, "The Kubernetes namespace to manage. "+
 		"Will manage custom resource objects of the managed CRD types for the whole cluster if unset.")
+
+	enableMetrics = flag.Bool("enable-metrics", false, "Whether to enable the "+
+		"metrics endpoint.")
+	metricsPort     = flag.String("metrics-port", ":10254", "Port for the metrics endpoint.")
```

Comment:
https://stackoverflow.com/questions/28322997/how-to-get-a-list-of-values-into-a-flag-in-golang